### PR TITLE
[ci skip] Added localhost IPv6

### PR DESCRIPTION
**Pull request by sharmaansh21**

### CODE REVIEW

File: actionpack/lib/action_dispatch/http/request.rb [hunk 1]
```diff
@@ -358,7 +358,7 @@ def authorization
       get_header('REDIRECT_X_HTTP_AUTHORIZATION')
     end
 
-    # True if the request came from localhost, 127.0.0.1.
+    # True if the request came from localhost, 127.0.0.1, ::1.
```
Comment (eileencodes) [inline actionpack/lib/action_dispatch/http/request.rb]: Thanks for the PR! :smile:

Can you update this to say "True if the request came from localhost, 127.0.0.1, or ::1." to be more grammatically correct?

**Comment by justanshulsharma** [comment]
@eileencodes :+1:

**Comment by eileencodes** [comment]
Thanks for the quick update!

**Comment by justanshulsharma** [comment]
@eileencodes welcome

**Pull request merged**